pkg/synthetictests/allowedalerts/watchdog.go

```diff
@@ -73,6 +76,36 @@ func IsWatchdogAlert(eventInterval monitorapi.EventInterval) bool {
 }
 
 func (a *watchdogAlertTest) InvariantCheck(ctx context.Context, restConfig *rest.Config, alertIntervals monitorapi.Intervals) ([]*junitapi.JUnitTestCase, error) {
+
+	// Skip this test when SNO is being upgraded
```

Comment:
where are you checking that the cluster was upgraded, not just a regular test?  YOu need to see the clusterversion history, right?

Reply:
is it only cluster upgrades when we'd want to bypass this?  anything that triggers a reboot or reschedule would cause this to fail
(but even if that is true, the comment should reflect that then)

Reply:
Ah, I assumed disruption tests run only during upgrade. Added cluster version history check to be sure